README.md

```diff
@@ -43,8 +43,8 @@ And return `nil`.
 
 1. Fork and clone this lab. 
 2. Open it in your text editor and run the test suite. You'll be coding your solution in `countdown.rb`
-1. Write a method that returns "HAPPY NEW YEAR!" that takes in an argument of an integer and uses a while loop to countdown from that integer to 0. 
-	* A method will return the very last line of code that it executes. You'll want to be able to pass any infinitely large number to your method as an argument and have it count down. Remember to use interpolation in creating your return value. You'll also want your loop to print out the "#{number} SECONDS!"
+1. Write a method that returns "0 left. HAPPY NEW YEAR!" that takes in an argument of an integer and uses a while loop to countdown from that integer to 0. 
```

Comment:
Hey @aenbar this lab is not asking students to return "0 left. HAPPY NEW YEAR!" The return value of the function described here should just return "HAPPY NEW YEAR!" this is to show the difference between printed values (like "x" left.") and the actual return value of the function